feature-extensions/multiaccount/README.md

```diff
@@ -0,0 +1,102 @@
+
+## Prerequisites
+- Make sure you have an admin AWS account and it will be used while installing the feature extension for multiaccount
+- Installer will use the AWS credentials that you have configured in your system by running 'aws configure'
+
+## Basic Usage
+
+```sh
+cd JAZZ_INSTALLER_DIRECTORY/feature-extensions/multiaccount
+python setup.py [-h] [--aws-region AWS_REGION]
+                [--jazz-stackprefix JAZZ_STACKPREFIX] [--aws-accesskey AWS_ACCESSKEY]
+                [--aws-secretkey AWS_SECRETKEY] [--jenkins-url JENKINS_URL]
+                [--jenkins-username JENKINS_USERNAME]
+                [--jenkins-password JENKINS_PASSWORD]
+                [--jazz-username JAZZ_USERNAME]
+                [--jazz-password JAZZ_PASSWORD]
+                [--jazz-apiendpoint JAZZ_APIENDPOINT]
+                {install} ...
+```
+Note:
+   - JAZZ_INSTALLER_DIRECTORY - Directory path of jazz installation
+   - Install option will ask user for raw inputs for all the missing arguments (similar to below)
+```sh
+    Please enter the environment prefix you used for your Jazz install:
+    Enter AWS accesskey of the new account:
+    Enter secretkey of the new account:
+    Enter AWS regions with space delimiter:
+    Please enter the Jenkins URL(without http):
+    Please enter the Jenkins Username:
+    Please enter the Jenkins Password:
+    Please enter the Jazz Admin Username:
+    Please enter the Jazz Admin Password:
+    Please enter the Jazz API Endpoint(Full URL):
+```
+
+## Example (Install option)
+
+### With all the parameters as arguments
+
+```sh
+cd JAZZ_INSTALLER_DIRECTORY/feature-extensions/multiaccount
+python setup.py --aws-region us-east-1 us-west-2
+                --jazz-stackprefix STACKPREFIX
+                --aws-accesskey XXXXX
+                --aws-secretkey XXXXX
+                --jenkins-url JENKINSURL
+                --jenkins-username JENKINSUSER
+                --jenkins-password JENKINSPASSWORD
+                --jazz-username JAZZADMIN
+                --jazz-password JAZZPASSWORD
+                --jazz-apiendpoint JAZZ_APIENDPOINT install
+
+```
+
+### With missing parameters as arguments ###
+
+```sh
+cd JAZZ_INSTALLER_DIRECTORY/feature-extensions/multiaccount
+python setup.py install
+
+# Enter the missing input params
+Please enter the environment prefix you used for your Jazz install:
+Enter AWS accesskey of the new account:
+Enter secretkey of the new account:
+Enter AWS regions with space delimiter:
+Please enter the Jenkins URL(without http):
+Please enter the Jenkins Username:
+Please enter the Jenkins Password:
+Please enter the Jazz Admin Username:
+Please enter the Jazz Admin Password:
+Please enter the Jazz API Endpoint(Full URL):
+```
+
+## Example (Uninstall option)
+
+### With all the parameters as arguments
+
+```sh
+cd JAZZ_INSTALLER_DIRECTORY/feature-extensions/multiaccount
+python triggerJenkinsDeleteResources.py --jenkins-url JENKINSURL
```

Comment:
If this is an independent script, can we rename it as `teardown.py`? `triggerJenkinsDeleteResources` looks too detailed and can be replaced with simple name (similar to `setup`).

Reply:
it shouldn't be a separate script, like the main installer, the syntax should be `installer.py uninstall <opts>`

Single entry point for install/uninstall.

Reply:
> If this is an independent script, can we rename it as `teardown.py`? `triggerJenkinsDeleteResources` looks too detailed and can be replaced with simple name (similar to `setup`).

Yes.

Reply:
> it shouldn't be a separate script, like the main installer, the syntax should be `installer.py uninstall <opts>`
> 
> Single entry point for install/uninstall.

Sure, will create a new followup PR for that.